src/Microsoft.Azure.Functions.Worker.Extensions.Mcp/McpToolPropertyAttribute.cs

```diff
@@ -3,10 +3,12 @@
 
 using Microsoft.Azure.Functions.Worker.Converters;
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Extensions.Mcp.Converters;
 
 namespace Microsoft.Azure.Functions.Worker.Extensions.Mcp;
 
-[ConverterFallbackBehavior(ConverterFallbackBehavior.Default)]
+[InputConverter(typeof(TypeConverter))]
```

Comment:
We also need to make sure this conversion applies to the scenarios currently handled by binding data conversion (e.g., all the properties defined using the bootstrapping model). It's possible that would be a different converter altogether, but the scenario needs to be handled.